lib/taurus/qt/qtgui/input/qwheel.py

```diff
@@ -893,6 +963,13 @@ def setNAN():
 
     def setNone():
         arrowWidget.setValue(None)
+
+    def disableHideEditWidget():
```

Comment:
Just a minor thing: I personally would either remove the `disableHideEditWidget` and `enableHideEditWidget` methods or make the `setHideEditWidget` "protected" (i.e. rename it to `_setHideEditWidget`). This is to avoid confusion on what is the recomended API: [*"There should be one-- and preferably only one --obvious way to do it."*](https://www.python.org/dev/peps/pep-0020/))

Reply:
> It is not inside the class but in main function

Ups! I missed that. Sorry. You are 100% right. There is no need toworry about the implementation in the `main()` function